packages/ui-react-rx/src/util/balanceFormat.ts

```diff
@@ -5,7 +5,7 @@
 import BN from 'bn.js';
 import { UInt } from '@polkadot/types/codec';
 
-import decimalFormat from './decimalFormat';
+import { decimalFormat } from './index';
```

Comment:
This one I would prefer as a direct import, since this file itself is in the index export.